src/hyp3_isce2/burst.py

```diff
@@ -337,3 +338,24 @@ def get_product_name(
         The name of the interferogram product.
     """
     return f'{reference_scene}x{secondary_scene}'
+
+
+def search_cmr_uat(scene_name: str) -> asf_search.ASFSearchResults:
+    opts = asf_search.ASFSearchOptions(host='cmr.uat.earthdata.nasa.gov')
+    return asf_search.search(product_list=[scene_name], opts=opts)
+
+
+def get_burst_params(scene_name: str) -> BurstParams:
+    results = search_cmr_uat(scene_name)
+
+    if len(results) == 0:
+        raise ValueError(f'ASF Search failed to find {scene_name}.')
+    if len(results) > 1:
+        raise ValueError(f'ASF Search found multiple results for {scene_name}.')
```

Comment:
Not blocking, but personally I think this may be better as a single if statement that includes the number of results. Like:
```suggestion
    if len(results) != 1:
        raise ValueError(f'ASF Search found {len(results)} results for {scene_name}.')
```
Then both the tests would have regexes matching `found 0 results` and `found 2 results`.

Reply:
100% agree. But not going to act on it now because I don't want to change the tests.